source/docs/casper/dapp-dev-guide/sdkspec/types_chain.md

```diff
@@ -20,7 +20,7 @@ Required Parameters:
 
 ## AccountHash {#accounthash}
 
-Hex-encoded Account hash.
+The AccountHash is a Hex-encoding of the PublicKey and therefore can be viewed as an Address.
```

Comment:
I noticed that uppercase words are used when referencing types that are listed on this page, e.g. Account, Bid, Deploy, but **address** is not among them.

I suggest either:
```diff
- The AccountHash is a Hex-encoding of the PublicKey and therefore can be viewed as an Address.
+ The AccountHash is a Hex-encoding of the PublicKey and therefore can be viewed as an address.
```
or 
```diff
- The AccountHash is a Hex-encoding of the PublicKey and therefore can be viewed as an Address.
+ The AccountHash is a Hex-encoding of the PublicKey and therefore can be viewed as an Account address.
```

Reply:
i pushed a new commit that should resolve this. 💯